clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/caching/agent/KubernetesCacheDataConverter.java

```diff
@@ -371,6 +371,28 @@ static void addSingleRelationship(Map<String, Collection<String>> relationships,
     return result;
   }
 
+  static List<CacheData> getClusterRelationships(String account, CacheData cacheData) {
```

Comment:
Can this list ever have more than 1 entry? If not, I would use a `Optional` here instead.

Reply:
I know it makes the stream below a feel little nicer with the flatmap -- but I think for the sake of consistency having this behave more like the above `convertAs*` methods (which are nullable rather than optional, that works too) will make this more readable/understandable in the future I think.

Reply:
Makes sense, updated!